examples/no-repeated-options.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// How might I throw if an option is repeated?
+
+// 1. const { parseArgs } = require('node:util'); // from node
+// 2. const { parseArgs } = require('@pkgjs/parseargs'); // from package
+const { parseArgs } = require('..'); // in repo
+
+const options = {
+  ding: { type: 'boolean', short: 'd' },
+  beep: { type: 'boolean', short: 'b' }
+};
+const { values, tokens } = parseArgs({ options, tokens: true });
+
+const seenBefore = new Set();
+const repeatedToken = tokens
+  .filter((t) => t.kind === 'option')
+  .find((t) => {
+    if (seenBefore.has(t.name)) return true;
```

Comment:
Side-effecting functions in `filter` or `find` is a nice trick, but I'm worried it might be confusing for readers who haven't encountered the pattern before. It might be clearer to use an explicit loop:

```js
for (const token of tokens) {
  if (token.kind !== 'option') continue;
  if (seenBefore.has(token.name)) {
    throw new Error(`option '${repeatedToken.name}' used multiple times`);
  }
  seenBefore.add(token.name);
}
```

Reply:
Used a forEach, but otherwise updated as suggested. Thanks.